linetools/isgm/abscomponent.py

```diff
@@ -396,6 +396,10 @@ def synthesize_colm(self, overwrite=False, redo_aodm=False, **kwargs):
         # Collate
         self.flag_N = 0
         for aline in self._abslines:
+            if aline.attrib['flag_N'] == 0:  # No value
+                warnings.warn("Absline {} has flag=0.  Hopefully you expected that")
```

Comment:
Did you mean to print the absorption line info here? There is a '{}' but no argument to print.